### DISCUSSION TITLE
How Can I Contribute To This Project?

### DISCUSSION THREAD

**Discussion by avciberkay** [zoelesv/Smathchat#5] [Q&A] [answered]
Hi, I am just a beginner on GitHub. I created my account 5 days ago and I found this project immediately. I want to be a part of this project. Can you please tell me what I can do for the project? Can you give me some simple tasks to do as a start? I would really appreciate if you could help me.

**Comment by zoelesv** [comment, marked as answer]
Hi there! Welcome to Github!!

With Smathchat, we are generating more mental math challenges for users to solve. Feel free to fork it and create new code.
At the same time, we want to deploy it on a different platform other than  Heroku because Heroku has an unresolved web socket problem. These are two main issues we are working on.
If you have any suggestions or questions let me know.

Happy coding!
